sdk/metricsadvisor/ai-metrics-advisor-rest/src/models.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Represents an alert reported by Metrics Advisor service.
+ */
```

Comment:
I don't think we need to define custom models, we should use that was generated

Reply:
removed